ci-operator/step-registry/medik8s/sbr/nfs-bastion/medik8s-sbr-nfs-bastion-commands.sh

```
@@ -0,0 +1,250 @@
+# update CRI-O config via a path that does not change the rendered MachineConfig name.
+# Active verification is the only reliable signal that nodes have picked up the mirror.
+log "Waiting for ubi8/ubi-minimal to be pullable via ITMS mirror (up to 12m)..."
+UBI_PULLABLE=false
+for i in $(seq 1 24); do
+    sleep 30
+    oc delete pod sbr-ubi-preflight --ignore-not-found --wait=false 2>/dev/null || true
+    POD_PHASE=$(oc run sbr-ubi-preflight \
+        --image=registry.access.redhat.com/ubi8/ubi-minimal:latest \
+        --restart=Never \
+        --command -- sh -c 'echo PREFLIGHT_OK' 2>/dev/null && \
+        timeout 30 bash -c 'until [[ $(oc get pod sbr-ubi-preflight -o jsonpath="{.status.phase}" 2>/dev/null) =~ ^(Succeeded|Failed)$ ]]; do sleep 2; done; oc get pod sbr-ubi-preflight -o jsonpath="{.status.phase}"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)